MAUI/Themes/Keys.md

```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
+        </td>
+        <td>
+            Background color of image editor.
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+    <tr>
+    <td>
+            SfImageEditorNormalToolbarBackgroundColor     
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of toolbar background.
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfImageEditorNormalToolbarBorderColor      
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of toolbar border in image editor. 
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfImageEditorNormalToolbarIconColor       
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of toolbar icon in image editor. 
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfImageEditorDisabledToolbarIconColor        
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of toolbar icon when it is disabled.  
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfImageEditorHoverToolbarIconBackground         
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of toolbar icon when it is hovered.   
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfImageEditorPressedToolbarIconBackground          
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of toolbar icon when it is in pressed state. 
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfImageEditorNormalToolbarTextColor           
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of toolbar text in image editor.   
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfImageEditorHoverToolbarTextColor            
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of toolbar text when it is hovered.   
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfImageEditorPressedToolbarTextColor             
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of toolbar text when it is pressed.    
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfImageEditorDisabledToolbarTextColor              
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of toolbar text when it is disabled.    
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfImageEditorTooltipBackground               
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of tooltip.     
```

Comment:
Background Color of tooltip in Image editor toolbar

Reply:
Updated